src/ntapfuse_ops.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * Project: ntapfuse
+ * Author: Samuel Kenney <samuel.kenney48@gmail.com>
```

Comment:
Since you are editing this file, please change the authors. The same goes for other files.